src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -681,7 +683,12 @@ class IrsPlan extends React.Component<
     if (e && e.currentTarget && e.currentTarget.id) {
       // update map position
       const clickedTableCrumb = this.state.tableCrumbs.find(c => c.id === e.currentTarget.id);
-      if (clickedTableCrumb && clickedTableCrumb.bounds && this.state.country) {
+      const { newPlan } = this.state;
```

Comment:
Why are we calling this "newPlan"?

Reply:
It is the plan being worked on that that particular time. Maybe it should have been named 'currentPlan'

Reply:
yes please